hasher-matcher-actioner/Makefile

```diff
@@ -0,0 +1,47 @@
+.PHONY: all docker upload_docker dev_create_configs dev_create_instance dev_destroy_instance dev_clear_configs clean
+
+shell-or-die = $\
```

Comment:
Could you add a comment explaining this command?  (even if it is ~common make/shell magic)